_data/simple-icons.json

```diff
@@ -14682,6 +14682,11 @@
             "hex": "7D00FF",
             "source": "https://www.stellar.org/press"
         },
+        {
+            "title": "Stencil",
+            "hex": "000000",
```

Comment:
Looking at their [website](https://stenciljs.com/), I think I would opt for something like `#5530FF`. The logo on their [Github page](https://github.com/ionic-team/stencil) also have a purplish color.